### ISSUE TITLE
How to run inference on cpu?

### ISSUE THREAD

**Issue by afrin321** [haotian-liu/yolact_edge#145]
Hello, can yolact_edge inference be run on cpu with saved weights?

I tried to follow the colab notebook code in jupyter notebook. 
It is giving an error saying "Torch not compiled with Cuda enabled.

Please help.
Thank you.

**Comment by haotian-liu** [comment]
Yes, but you need to modify the code and make sure all the tensors / model weights are on the CPU.
